### PR TITLE
Add GPG GUI menu

### PR DESCRIPTION
**Pull request by kylerankin**
It makes more logical sense for GPG functions to be split out into their
    own menu instead of being part of the "Flash" menu. This creates a
    gpg-gui.sh script and moves GPG options there while adding a few
    additional features (like listing keys and initial smartcard key
    generation support).

### CODE REVIEW

File: initrd/bin/gpg-gui.sh [hunk 1]
```diff
@@ -0,0 +1,244 @@
+#!/bin/sh
+#
+set -e -o pipefail
```
Comment (tlaurion) [inline initrd/bin/gpg-gui.sh]: The effect of this is that once one of the command in a pipe fails, the whole line fails.

**Comment by kylerankin** [comment]
@tlaurion I've removed the pipefail statement from gpg-gui.sh.

**Comment by tlaurion** [comment]
@kylerankin : It was my error. You did checks on file presence, which doesn't cause any problems. My attempts to do a while loop on the non presence of a match string was causing the issue. 

Setting pipefail is debattable, but was not causing any issue in your PR.
If you want to put it back in, there is no problem.

Please rebase on master so I can merge. :)

File: initrd/bin/gpg-gui.sh [hunk 2]
```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 #
-set -e -o pipefail
```
Comment (tlaurion) [inline initrd/bin/gpg-gui.sh]: Please add it back in.

**Comment by kylerankin** [comment]
Changed back.

**Pull request merged**